### DISCUSSION TITLE
Display driver for 1-2-3

### DISCUSSION THREAD

**Discussion by rruhle** [taviso/123elf#113] [General]
In May of 2022, The Register published an article regarding Tavis' work with 123 on Linux.
https://www.theregister.com/2022/05/25/lotus_123_for_linux_appears/

That article also describes Tavis' effort enabling support for more than an 80×25 window.

Was this display driver built into the current release, or is there another way to activate it?

**Comment by taviso** [comment]
Yes, it's already in UNIX version, you can use any terminal size - if you want to use it for the DOS version, it's here:

https://github.com/taviso/lotusdrv

I only use the UNIX version now, so I haven't updated it for a long time 😄
